src/Discord.Net.Rest/Entities/Interactions/RestGuildMessageCommand.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Model = Discord.API.ApplicationCommand;
+
+namespace Discord.Rest
+{
+    /// <summary>
+    ///     Represents a Rest-based guild command.
+    /// </summary>
+    public class RestGuildMessageCommand : RestApplicationCommand
```

Comment:
As now there are more guild commands, wouldn't it be better to have interface or base class for guild commands that would have `GetGuild` method? That way underlying guild of command could be easier to get from yet unknown type of command instead of checking for Message, User, Slash commands

Reply:
I agree with this

Reply:
So, duplicate what i did with the SocketCommandBase, But as a RestGuildCommandBase, just to implement the GetGuild method?